src/main/resources/log4j2-json.properties

```diff
@@ -0,0 +1,34 @@
+status = info
```

Comment:
Architects updated doc with recommended configuration: https://wiki.folio.org/display/DD/Folio+logging+solution 
Based on this we should set status  = error and delete dest property

Reply:
Ok, I have got it. Should I also add threshold part
![image](https://user-images.githubusercontent.com/113523904/207612586-aa07e931-065e-4d2a-acf0-1283f51fe9a8.png)

Reply:
We have not added it for other modules also, but I think we are okay without it